sample/src/main/java/io/ona/kujaku/sample/activities/MainActivity.java

```diff
@@ -165,6 +206,14 @@ private void fetchDroppedPoints(OnFinishedListener onFinishedListener) {
         genericAsyncTask.executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
     }
 
+    private void setMapDownloadStop(boolean enabled) {
+        if (mapDownloadStop != enabled) {
```

Comment:
What is this line meant to achieve?

Reply:
Not have to call `stopMapDownloadBtn.setVisibility(enabled ? View.VISIBLE : View.GONE);` repetitively if the desired state is current.

Views can be unreactive to events if you you update them too frequent.

I expect map progress updates to be frequent

Reply:
Cool. Please add this as a code comment.